src/core/client/admin/routes/Moderate/SingleModerate/SingleModerate.css

```diff
@@ -1,11 +1,15 @@
+$single-moderate-background: var(--v2-colors-grey-100);
```

Comment:
we don't have designs for the single comment moderate view but i just updated colours + fonts.

Reply:
We can work with Sam to get these specc'ed and updated in another ticket. This is good for now. :)